### PR TITLE
[New Hack] - 069 Fabric Real Time Analytics

### PR DESCRIPTION
**Pull request by CameronKahrs-Microsoft**

### CODE REVIEW

**Review by skydiver1980 — APPROVED** [review]

File: 069-FabricRealTimeAnalytics/Coach/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# What The Hack - Fabric Real-time Analytics - Coach Guide
+
+## Introduction
+
+Welcome to the coach's guide for the Fabric Real-time Analytics What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+
+This hack includes an optional [lecture presentation](Lectures.pptx) that features short presentations to introduce key topics associated with each challenge. It is recommended that the host present each short presentation before attendees kick off that challenge.
```
[the remark this answers is not in the document]
Comment (CameronKahrs-Microsoft) [reply 069-FabricRealTimeAnalytics/Coach/README.md]: removed.

File: 069-FabricRealTimeAnalytics/Coach/Solution-00.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+**[Home](./README.md)** - [Next Solution >](./Solution-01.md)
+
+## Introduction
+
+Thank you for participating in the Fabric real time analytics WTH. Before you can hack, you will need to set up some prerequisites.
+
+## Common Prerequisites
+
+- [Azure Subscription](../../../000-HowToHack/WTH-Common-Prerequisites.md#azure-subscription)
```
[the remark this answers is not in the document]
Comment (CameronKahrs-Microsoft) [reply 069-FabricRealTimeAnalytics/Coach/Solution-00.md]: links fixed. had an extra ../ file up command

**Review by Whowong — CHANGES_REQUESTED** [review]
Everything else looks great, I am just having an issue with this link resolving

File: 069-FabricRealTimeAnalytics/Student/Challenge-00.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Description
+
+Now that you have the common pre-requisites installed on your workstation, there are prerequisites specific to this hack.
+
+Your coach will provide you with a Resources.zip file that contains resources you will need to complete the hack. If you plan to work locally, you should unpack it on your workstation. If you plan to use the Azure Cloud Shell, you should upload it to the Cloud Shell and unpack it there.
+
+Please install these additional tools:
+
+- [Azure IoT Tools](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.azure-iot-tools) extension for Visual Studio Code
```
[the remark this answers is not in the document]
Comment (CameronKahrs-Microsoft) [reply 069-FabricRealTimeAnalytics/Student/Challenge-00.md]: yeah the link is dead. this one was from the premade stuff. I changed the link to the documentation for installing the visual studio code extension

**Comment by CameronKahrs-Microsoft** [comment]
all changes requested have been resolved

**Review by Whowong — APPROVED** [review]

**Review by perktime — APPROVED** [review]

**Review by perktime — APPROVED** [review]

**Comment by perktime** [comment]
All requested changes seem to have been addressed

**Pull request merged**